### PR TITLE
molden: migrate from homebrew/science

### PR DESCRIPTION
**Pull request by fxcoudert**
`molden` has 54 installs in the last 90 days.

### CODE REVIEW

File: Formula/molden.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  url "ftp://ftp.cmbi.ru.nl/pub/molgraph/molden/molden5.7.tar.gz"
+  sha256 "5e7b3a8bf9251626d362b1d1dd8e4362054e6bce505c43e10254265857bb6e7d"
+  revision 1
+
+  depends_on :fortran
+  depends_on :x11
+
+  def install
+    system "make"
+    bin.install "molden", "gmolden"
```
Comment (ilovezfs) [inline Formula/molden.rb]: Needs a `make install` to be considered for inclusion.

File: Formula/molden.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  test do
+    # molden is an interactive program, there is not much we can test here
+    assert_match "Molden#{version}", shell_output("#{bin}/gmolden -h")
```
Comment (ilovezfs) [inline Formula/molden.rb]: Needs a real test

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/molden.rb (hunk 2 above)
Comment (fxcoudert) [reply Formula/molden.rb]: There is no way for molden not to open the GUI and expect user interaction (apart from the admittedly useless `-h`). Another option is to run `otool` on the binary and check that it is linked appropriately?

File: Formula/molden.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/molden.rb]: The makefile uses `sudo` and hardcodes `/usr/local/bin`. I've reported it by email, but it seems cleaner to install binaries than to patch.

Comment (ilovezfs) [reply Formula/molden.rb]: OK, I think we'll need to pass on this PR then.

Comment (fxcoudert) [reply Formula/molden.rb]: 1227 formulas have `bin.install`, and it's not only legacy stuff. We regularly accept new formulas with manual install for Rust and other non make-based projects. Many of our top formulas use it, such as go.

Comment (ilovezfs) [reply Formula/molden.rb]: This uses make, and needs a functional make install to be included. The way cargo works is irrelevant.

**Pull request closed without merging**